build.sbt

```diff
@@ -30,23 +30,19 @@ resolvers += Resolver.jcenterRepo
 
 libraryDependencies += guice
 libraryDependencies += "org.scalatestplus.play" %% "scalatestplus-play" % "3.1.2" % Test
-
-val silhouetteVersion = "5.0.0"
-libraryDependencies ++= Seq(
-  "com.mohiva" %% "play-silhouette" % silhouetteVersion,
-  "com.mohiva" %% "play-silhouette-persistence" % silhouetteVersion,
-  "com.mohiva" %% "play-silhouette-crypto-jca" % silhouetteVersion,
-  "com.mohiva" %% "play-silhouette-password-bcrypt" % silhouetteVersion
-)
-
-
+libraryDependencies += "com.h2database" % "h2" % "1.4.196"
```

Comment:
Why do we need `h2`?
Please remove if not necessary. This version is vulnerable anyways.

Reply:
@johannesduesing according to git you added this dependency. can we remove it, or why did you add it?

Reply:
Since I don't recall using this h2 database I guess it was either Auto-inserted when I created the initial project or part of the silhouette example project I studied to integrate silhouette. Either way, should be safe to remove it here.